tests/test_scripts.py

```diff
@@ -65,34 +63,37 @@ def test_eval_policy(config):
       assert isinstance(run.result, dict)
 
 
-def test_train_adversarial():
+@pytest.mark.parametrize("plots", [True, False])
+def test_train_adversarial(plots: bool):
   """Smoke test for imitation.scripts.train_adversarial"""
-  with tempfile.TemporaryDirectory(prefix='imitation-train',
-                                   ) as tmpdir:
-      config_updates = {
-          'init_trainer_kwargs': {
-              # Rollouts are small, decrease size of buffer to avoid warning
-              'trainer_kwargs': {
-                  'n_disc_samples_per_buffer': 50,
-              },
-          },
-          'log_root': tmpdir,
-          'rollout_glob': "tests/data/rollouts/CartPole*.pkl",
-      }
-      run = train_ex.run(
-          named_configs=['cartpole', 'gail', 'fast'],
-          config_updates=config_updates,
-      )
-      assert run.status == 'COMPLETED'
+  with TemporaryDirectory(prefix='imitation-train') as tmpdir:
+    named_configs = ['cartpole', 'gail', 'fast']
+    if plots:
```

Comment:
Why not always test with plots? Are they any code paths not tested with plots that are tested without?

Reply:
No doesn't look like it. I'm removing this now.